lib/quadrigacx/client/private.rb

```diff
@@ -49,23 +49,24 @@ def cancel params={}
       request(:post, '/cancel_order', params)
     end
 
+    # Return a JSON list of open orders.
+    #
+    # book - optional, if not specified, will default to btc_cad.
+    def open_orders params={}
+      request(:post, '/open_orders', params)
+    end
+
     # Withdraw bitcoins.
     #
     # amount  – The amount to withdraw.
     # address – The bitcoin address we will send the amount to.
-    def withdraw params={}
+    def bitcoin_withdraw params={}
```

Comment:
Do you think Quadriga will have many more coin types in the future? Might be worth making the coin type a param. I'm mostly indifferent though.

Reply:
At this point, I'm just going to get the main three coins as their own methods. I do have the same feelings as you though. It might make sense to DRY this up by passing in a coin type (or even metaprogramming to create methods based on coin types).